save-backend/src/main/kotlin/com/saveourtool/save/backend/repository/LnkContestProjectRepository.kt

```diff
@@ -23,66 +23,45 @@ interface LnkContestProjectRepository : BaseEntityRepository<LnkContestProject>
     fun findByProject(project: Project): List<LnkContestProject>
 
     /**
-     * @param projectId
-     * @return List of [LnkContestProject] by [projectId]
-     */
-    fun findByProjectId(projectId: Long): List<LnkContestProject>
-
-    /**
-     * @param contestId
-     * @return List of [LnkContestProject] by [contestId]
-     */
-    fun findByContestId(contestId: Long): List<LnkContestProject>
-
-    /**
-     * @param contestId
-     * @param projectId
-     * @return [LnkContestProject] by [contestId] and [projectId]
-     */
-    fun findByContestIdAndProjectId(contestId: Long, projectId: Long): Optional<LnkContestProject>
-
-    /**
-     * @param projectIds
-     * @return list of [LnkContestProject] in which [Project]s have ids from [projectIds]
+     * @param contest
+     * @param project
+     * @return [LnkContestProject] by [contest] and [project]
      */
-    fun findByProjectIdIn(projectIds: Set<Long>): List<LnkContestProject>
+    fun findByContestAndProject(contest: Contest, project: Project): Optional<LnkContestProject>
 
     /**
-     * @param contestName
      * @param projectName
+     * @param organizationName
+     * @param pageable
      * @return link contest-project for project with name [projectName] and contest with name [contestName]
      */
-    fun findByContestNameAndProjectName(contestName: String, projectName: String): Optional<LnkContestProject>
+    fun findByProjectNameAndProjectOrganizationName(
+        projectName: String,
+        organizationName: String,
+        pageable: Pageable,
+    ): List<LnkContestProject>
 
     /**
      * @param contestName
      * @return list of [LnkContestProject] linked to contest with name [contestName]
      */
     fun findByContestName(contestName: String): List<LnkContestProject>
 
-    /**
-     * @param project
-     * @param page
-     * @return list of best contests of the project
-     */
-    fun findByProjectOrderByScoreDesc(project: Project, page: Pageable): Page<LnkContestProject>
-
     /**
      * Save [LnkContestProject] using only ids and contest score.
      *
      * @param contestId
      * @param projectId
-     * @param score
+     * @return saved [LnkContestProject] record
      */
     @Transactional
     @Modifying
     @Query(
-        value = "insert into save_cloud.lnk_contest_project (project_id, contest_id, score) values (:project_id, :contest_id, :score)",
+        value = "insert into save_cloud.lnk_contest_project (project_id, contest_id) values (:project_id, :contest_id)",
```

Comment:
why you don't use standard `save()` method?

Reply:
actually, I see that you use standard `save` method, why we need this one?

Reply:
Well, we should use a standard one, will fix it